lib/Service/FoldersFilter.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @author Baptiste Fotia <baptiste.fotia@hotmail.com> for Arawa (https://arawa.fr)
+ *
+ * GroupFolders
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\GroupFolders\Service;
+
+use OCP\IConfig;
+use OCP\IUserSession;
+use OCP\IGroupManager;
+
+class FoldersFilter {
+
+	/** @var IUserSession */
+	private $userSession;
```

Comment:
Use php 7.4 props instead of phpdoc

Reply:
I fixed this part in this commit : https://github.com/nextcloud/groupfolders/pull/2072/commits/71eb35975796624c11fc280c6290ef8ab49209f9